ts/features/bonus/cdc/screens/CdcBonusRequestInformationTos.tsx

```diff
@@ -77,7 +77,8 @@ const CdcBonusRequestInformationTos = () => {
             )}
           </View>
           <VSpacer size={16} />
-          <IORenderHtml
+          {/* Replace the following chunk of code with IOMarkdown */}
```

Comment:
Until it is replaced, what will be displayed in its place?

Reply:
@ChrisMattew I might sound a bit cynical, but it doesn't matter because `CdC` has been a frozen project for a long time. I had to remove `IORenderHtml` because it had references to the legacy styles, and fortunately now we have a very good replacement. If we have to revive the project, every single screen will be checked again anyway.